tests/Framework/OmnipayBlueSnapTestCase.php

```diff
@@ -7,10 +7,10 @@
 use Guzzle\Common\Event;
 use Guzzle\Http\Message\Response;
 
-class TestCase extends \Omnipay\Tests\TestCase
+class OmnipayBlueSnapTestCase extends \Omnipay\Tests\TestCase
```

Comment:
why the rename? it's already in omnipay-bluesnap, so seems redundant

Reply:
This had a naming collision with another class when travis was building and testing on PHP 5.x. This was not a problem for 7.x. I can rename it to something else if you'd like but it can't be reverted back to `TestCase`

Reply:
Here's a build failure of what I mean https://travis-ci.org/vimeo/omnipay-bluesnap/jobs/530455029

Reply:
weeeeeird. namespacing all looks correct. darn php 5. whatever